### PR TITLE
Temporarily disable Playwright to not block deployments

### PR DESCRIPTION
**Pull request by JCantu248**
…epared for full implementation.

#  #

## 🗣 Description ##

This is a temporary fix to skip Playwright UI testing for now and allow for deployments to continue.

## 💭 Motivation and context ##
This is needed to avoid blocking deployment.

## 🧪 Testing ##

Tested by verifying the pipeline stage was skipped on push and deployment.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by cduhn17 — APPROVED** [review]
LGTM

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -15,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: mcr.microsoft.com/playwright:v1.41.2-jammy
-    if: github.event.deployment_status.state == 'success'
+    if: true == false
```
Comment (dav3r) [inline .github/workflows/playwright.yml:19]: I highly recommend adding a comment here explaining why you are doing this, even if it's temporary, and when this change will be reverted.

**Review by dav3r — CHANGES_REQUESTED** [review]

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (JCantu248) [reply .github/workflows/playwright.yml:19]: Updated here: 3bf3a66362874d1d290b46327047781ae266b0f7

Comment (dav3r) [reply .github/workflows/playwright.yml:19]: You should also create an issue in this repo to go along with that TODO and mention the issue number in this comment (that is why we have this checkbox item in our PR template: "All future TODOs are captured in issues, which are referenced in code comments.")

Comment (JCantu248) [reply .github/workflows/playwright.yml:19]: The issue is being tracked in JIRA, not Github. 

The associated draft PR is found here: https://github.com/cisagov/XFD/pull/372 

...

Comment (dav3r) [reply .github/workflows/playwright.yml:19]: Excellent, that works! 👍

Comment (JCantu248) [reply .github/workflows/playwright.yml:19]: See commit: 704398f220fa676c3e6f5533c808a4d05e511b93

**Review by dav3r — APPROVED** [review]
👍

**Pull request merged**